plotsquared/plot-flags.md

```diff
@@ -0,0 +1,286 @@
+# Plot flags
+
+## Introduction
+
+The extensive flagging system allows you to do all sorts of things such as setting the plot weather.
+
+Setting a flag is easy, just use `/plot flag set <flag> <value>`. For a boolean flag, the value would either be 'true' or 'false'.
+
+## Flag List
+
+To bring up a list of flags use `/plot flag list`. These values are clickable, and clicking a flag name will show you information about that particular flag. You can also use `/plot flag info <flag>` directly, if you already know the flag name.
+
+![Flag system](https://i.imgur.com/uTPjJEE.png)
+
+## Default Plot and Road Flags
+
+You can set default flags per world in the `plugins/PlotSquared/config/worlds.yml`. Default flags will be present in each claimed plot unless a different value is manually set for that plot.
+
+The following is an **example** of setting the `use` flag to allow interaction with doors, and trapdoors, levers, gates, pressure plates, buttons and trip wires:
+
+```yaml
+worlds:
+  <world>:
+    [...]
+    flags:
+      use: "oak_door,iron_trapdoor,lever,acacia_fence_gate,light_weighted_pressure_plate,oak_button,tripwire"
+      pvp: true
+```
+
+_`<world>` indicates the name of that plotworld._
+
+You can use [Vanilla block tags](/plotsquared/vanilla-tags.md) too.
```

Comment:
Clicking on the hyperlink should link within the document on gitbook. Currently, it links to GitHub.